aws/services/S3/S3_Website_With_CloudFront_Distribution.yaml

```diff
@@ -69,8 +41,10 @@ Resources:
             HTTPPort: '80'
             HTTPSPort: '443'
             OriginProtocolPolicy: http-only
-          DomainName: !Join ['', [!Ref 'S3BucketForWebsiteContent', !FindInMap [Region2S3WebsiteSuffix,
-                !Ref 'AWS::Region', Suffix]]]
+          DomainName:
```

Comment:
Will this work? The mapping above uses the website-style DNS (.s3-website.eu-central-1.amazonaws.com) to serve the content. I thought the "DomainName" attribute returned the S3-style DNS (mystack-mybucket-kdwwxmddtr2g.s3.amazonaws.com).